.github/workflows/snapshot.yml

```diff
@@ -73,7 +71,7 @@ jobs:
           subject-path: "${{ github.workspace }}/**/target/*.jar"
 
       - if: ${{ failure() }}
-        uses: elastic/oblt-actions/slack/send@v1.2.0
+        uses: elastic/oblt-actions/slack/send@v1.5.0
```

Comment:
Maybe we can use `@v1` for actions  where we are sure that it won't have any effect on the build output.

Reply:
but if it's easier to just use the same version ref. than let's stick with it.

Reply:
I don't know yet.

If we need to do something to all the repositories we could use the `gh oblt` extension and apply the sam pattern.

For now, let's use the latest available version, and let dependabot drive the bumps